libpod/options.go

```diff
@@ -19,7 +20,7 @@ import (
 
 var (
 	nameRegex  = regexp.MustCompile("^[a-zA-Z0-9][a-zA-Z0-9_.-]*$")
-	regexError = errors.Wrapf(ErrInvalidArg, "names must match [a-zA-Z0-9][a-zA-Z0-9_.-]*")
+	regexError = errors.Wrapf(config2.ErrInvalidArg, "names must match [a-zA-Z0-9][a-zA-Z0-9_.-]*")
```

Comment:
Perhaps lack of tea, but should this be config and not config2?

Reply:
missed the config2 def in front at line 11.  But what is using config already here?

Reply:
the IDE does that when there is a conflict of some sort.